content/source/docs/enterprise/vcs/github-enterprise.html.md

```diff
@@ -78,23 +78,23 @@ The rest of this page explains the GitHub Enterprise versions of these steps.
 
     ![TFE screenshot: add client fields](./images/ghe-tfe-add-client-fields.png)
 
-3. Click "Create connection." This will take you back to the OAuth Configuration page, which now includes your new GitHub Enterprise client.
+3. Click "Create VCS Provider." This will take you back to the VCS Provider page, which now includes your new GitHub client.
```

Comment:
Hm, seems we should keep "Enterprise" here? Or is there some reason you removed it?

Reply:
That was a diffing error! Thanks for catching, I'll fix it.